tests/pipeline/test_pipeline.py

```diff
@@ -13,9 +16,15 @@
 
 
 class TestPipeline:
-    def test_load_pipeline(self, tmpdir, snapshot):
+    @pytest.fixture(autouse=True)
+    def setup_teardown(self) -> Iterator[None]:
+        # setup
         os.environ["KPOPS_ENVIRONMENT"] = "development"
-        output_file_path = tmpdir.join("pipeline.yaml")
+        yield  # testing
+        # teardown
+        del os.environ["KPOPS_ENVIRONMENT"]
```

Comment:
This is a bit strange. After the discussion [here](https://github.com/bakdata/kpops/pull/100#discussion_r1099787094). I have this feeling that all the values are read from the [config.yaml](https://github.com/bakdata/kpops/blob/main/config.yaml#L2) in the root directory.
IMO we don't need this at all since the mandatory `broker` address set is also not set by an env, but the tests run green. WDYT?

Reply:
true! `--config` already has a default set, so this is unnecessary

Reply:
Kill it with kindness hehehe